SRC/dgecon.f

```diff
@@ -106,6 +106,7 @@
 *>          INFO is INTEGER
 *>          = 0:  successful exit
 *>          < 0:  if INFO = -i, the i-th argument had an illegal value
+*>          =-5:  if ANORM is NAN.
```

Comment:
Maybe write "if ANORM is NaN or less than 0"

Reply:
I put "if ANORM is NaN or negative" because I like it better :)